github-orgs/kubeflow/org.yaml

```diff
@@ -27,6 +28,7 @@ orgs:
         has_repository_projects: true
         location: ""
         members:
+        - jbottum
         - tombuuz
```

Comment:
Can we order josh and tom alphabetically in the members list, so it is consistent?